lib/facebook.ex

```diff
@@ -375,6 +377,29 @@ defmodule Facebook do
     end
   end
 
+
+  @doc """
+  Build a stream resource from a paged facebook response
+
+  ## Examples
+    iex> stream = Facebook.pageFeed(:feed, "CocaColaMx", "<Your Token>", "id,name") |> Facebook.build_stream
```

Comment:
Yeah...I changed it from build_stream to buildStream to match the functions name style.